src/Plugins/SimplnxCore/docs/CropImageGeometry.md

```diff
@@ -15,6 +15,32 @@ Bounds: 100 voxels * 0.25 micron/voxel = 25 microns
 
 *Note:* The input parameters are *inclusive* and begin at *0*, so in the above example *0-99* covers the entire range of **Cells** in a given dimension.
 
+*Note on Physical Bounds:* If the physical coordinate bounds are outside the image geometry's bounds the nearest valid bound will be used. Example:
```

Comment:
I think we are going to need some more explicit documentation with images to fully explain what is going on.

Reply:
Documentation has been updated.